tests/storage/migration.go

```diff
@@ -296,6 +296,43 @@ var _ = SIGDescribe("[Serial]Volumes update with migration", Serial, func() {
 			waitForMigrationToSucceed(vm.Name, ns)
 		})
 
+		It("should migrate the source volume from a source and destination block RWX DVs", func() {
```

Comment:
So as you know, these kubevirt e2e tests get exercised on many other storage provisioners outside of this repository.
I wonder if us for example triggering a test run against trident would be equivalent to a dedicated test case for this scenario.

Just a thought ofc
``` 
{
    "storageClassRhel":        "trident-csi-fsx",
    "storageClassWindows":     "trident-csi-fsx",
    "storageRWXFileSystem":    "trident-csi-fsx",
    "storageRWOFileSystem":    "trident-csi-fsx",
    "storageSnapshot":         "trident-csi-fsx"
}
```
Or
```
{
    "storageClassRhel":        "ocs-storagecluster-ceph-rbd-virtualization",
    "storageClassWindows":     "ocs-storagecluster-ceph-rbd-virtualization",
    "storageRWXBlock":         "ocs-storagecluster-ceph-rbd-virtualization",
    "storageRWOFileSystem":    "ocs-storagecluster-ceph-rbd-virtualization",
    "storageRWOBlock":         "ocs-storagecluster-ceph-rbd-virtualization",
    "storageSnapshot":         "ocs-storagecluster-ceph-rbd-virtualization"
}
```

Reply:
@akalenyu not sure I understand the comment and suggestion. I have tested this with ceph and got the same result as described in https://github.com/kubevirt/kubevirt/issues/12811

Reply:
The issue was in kubevirt which wasn't setting correctly the storage copy options for the migration

Reply:
I am suggesting that we can avoid a specific functional test, and instead rely on the existing tests.
So given one of the config's above (which are real testing targets, btw, just not in kubevirt/kubevirt)
one of the existing tests would have caught this

Reply:
I would generally agree, but in this case the 2 tests exercise 2 different paths (in KubeVirt code) for RWX and RWO code. Hence, I'd like to keep the new test

Reply:
So we can do that just want to generally gauge if we could follow up by cutting the amount of tests here.
I was thinking that maybe if we used the dv.spec.storage API here more, these tests could exercise different paths when ran against different test suite configs. WDYT? does this make any sense?

Reply:
ping @alicefr do you think this is worth the effort? I can give it a go

Reply:
@akalenyu if we can exercise the different combinations, I'm fine. I have another bug (https://github.com/kubevirt/kubevirt/issues/12823) which also need to test the migration from block to fs. I would have also added a similar functional test for this as well

Reply:
FYI: https://github.com/kubevirt/kubevirt/pull/12851